cr5_robot:
  joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 50
  joint_controller:
    type: position_controllers/JointTrajectoryController
    joints:
      - joint1
      - joint2
      - joint3
      - joint4
      - joint5
      - joint6
    gains:
      joint1: {p: 1000.0, i: 0.0, d: 0.1, i_clamp: 0.0}
      joint2: {p: 1000.0, i: 0.0, d: 0.1, i_clamp: 0.0}
      joint3: {p: 1000.0, i: 0.0, d: 0.1, i_clamp: 0.0}
      joint4: {p: 1000.0, i: 0.0, d: 0.1, i_clamp: 0.0}
      joint5: {p: 1000.0, i: 0.0, d: 0.1, i_clamp: 0.0}
      joint6: {p: 1000.0, i: 0.0, d: 0.1, i_clamp: 0.0}
    constraints:
        goal_time: 0.6
        stopped_velocity_tolerance: 0.0
        joint1: {trajectory: 0.1, goal: 0.1}
        joint2: {trajectory: 0.1, goal: 0.1}
        joint3: {trajectory: 0.1, goal: 0.1}
        joint4: {trajectory: 0.1, goal: 0.1}
        joint5: {trajectory: 0.1, goal: 0.1}
        joint6: {trajectory: 0.1, goal: 0.1}
  stop_trajectory_duration: 0.5
  state_publish_rate:  25
  action_monitor_rate: 10
  
  gripper_controller:
    type: position_controllers/JointTrajectoryController
    joints:
      - gripper_finger1_joint
    gains:
      gripper_finger1_joint: {p: 50.0, i: 0.01, d: 1.0, i_clamp: 1.0}


